src/vs/base/browser/ui/button/button.css

```diff
@@ -69,6 +69,11 @@
 
 .monaco-button-dropdown > .monaco-button.monaco-dropdown-button {
 	border-left-width: 0 !important;
+	border-radius: 0 2px 2px 0;
```

Comment:
@lszomoru, I think with the changes in this file you can get rid of the border-radius in https://github.com/marrej/vscode/blob/f7e26dffdc2f78e8fb75610e5950791016ffcf74/src/vs/workbench/contrib/scm/browser/media/scm.css#L290-L293 and also https://github.com/marrej/vscode/blob/f7e26dffdc2f78e8fb75610e5950791016ffcf74/src/vs/workbench/contrib/scm/browser/media/scm.css#L295-L296